services/order.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic()
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+
+    order = Order.objects.create(user=user)
```

Comment:
it seems to me, that there is no need to create additional variable )
u can place ur get_user_model method inside an Order)
Also, u are using save method later, so u can use just Order for creating without objects manager